partcad-cli/src/partcad_cli/click/command.py

```
@@ -0,0 +1,156 @@
+import rich_click as click
+import sys
+import yaml
+import partcad as pc
+import coloredlogs, logging
```

Ruff (0.8.0)

Multiple imports on one line

Split imports

(E401)